src/main.rs

```diff
@@ -202,17 +203,26 @@ fn init_log() {
 #[cfg(not(feature = "logging"))]
 fn init_log() {}
 
-fn get_languages(env_lang: Option<&str>, env_language: Option<&str>) -> Vec<String> {
+fn get_languages(
+    command_line_lang: Option<&str>,
+    config_lang: Option<&str>,
+    env_lang: Option<&str>,
+    env_language: Option<&str>,
+) -> Vec<String> {
     // Language list according to
     // https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#language
 
-    if env_lang.is_none() {
-        return vec!["en".to_string()];
-    }
-    let env_lang = env_lang.unwrap();
+    let env_language = env_language.unwrap_or("").split(':');
 
-    // Create an iterator that contains $LANGUAGE (':' separated list) followed by $LANG (single language)
-    let locales = env_language.unwrap_or("").split(':').chain([env_lang]);
+    let locales: Vec<&str> = if let Some(command_line_lang) = command_line_lang {
+        vec![command_line_lang]
+    } else if let Some(config_lang) = config_lang {
+        vec![config_lang]
+    } else if let Some(env_lang) = env_lang {
+        env_language.chain(once(env_lang)).collect()
+    } else {
+        env_language.collect()
+    };
```

Comment:
I don't think this logic conforms to https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#language anymore, because the fast-path (`if env_lang.is_none() { return ... }`) was removed.

Reply:
Fixed.